lib/spack/spack/spec.py

```diff
@@ -750,9 +754,126 @@ def __str__(self):
         return "{deps: %s}" % ', '.join(str(d) for d in sorted(self.values()))
 
 
+def _libs_default_handler(descriptor, spec, cls):
+    """Default handler when looking for 'libs' attribute. The default
+    tries to search for 'lib{spec.name}' recursively starting from
+    `spec.prefix`.
+
+    :param ForwardQueryToPackage descriptor: descriptor that triggered
```

Comment:
Ah, just learned about the [Napoleon](http://www.sphinx-doc.org/en/stable/ext/napoleon.html#module-sphinx.ext.napoleon) extension for Sphinx which converts Google-style or Numpy-style docstrings to reStructuredText automatically. So I guess we could switch to Google-style eventually if we want. I agree that it is much more readable.